### DISCUSSION TITLE
Welcome to Jolie discussions

### DISCUSSION THREAD

**Discussion by fmontesi** [jolie/jolie#418] [Announcements]
Dear all,

I'm opening the GitHub discussions feature for this repository as an experiment. Some of our open-ended discussions require posting code and keeping track of comments, for which GitHub discussions might work well.

We're going to learn what this platform works best for by practice. Please feel free to use it for any kind of discussion for now, with the following exceptions in mind (this list might change over time):
- Issues and pull requests should still be discussed in their proper places: [issues](https://github.com/jolie/jolie/issues) and [pull requests](https://github.com/jolie/jolie/pulls).
- Questions and Answers on the use of Jolie should be kept to [Stack Overflow](https://stackoverflow.com/).
- Chatting should still be carried out on [our Discord server](https://discord.gg/yQRTMNX).

Enjoy! :)

**Comment by mwallnoefer** [comment]
I guess that the _Google Group Community_ has been closed meanwhile, but the README.MD still contains a reference to it:
> E-mail, by sending an e-mail to jolie-devel@googlegroups.com. Anybody can send to the group. You can also register to the e-mail group at this link: https://groups.google.com/g/jolie-devel.
